packages/apps-config/src/api/chain/index.ts

```diff
@@ -0,0 +1,9 @@
+// Copyright 2017-2022 @polkadot/apps-config authors & contributors
```

Comment:
```suggestion
// Copyright 2017-2023 @polkadot/apps-config authors & contributors
```

It took me some time to get to the PR (once 2023 hits CI actually checks against these)